src/pip/_internal/commands/install.py

```diff
@@ -71,54 +75,56 @@ def __init__(self, *args, **kw):
 
         cmd_opts.add_option(cmdoptions.editable())
         cmd_opts.add_option(
-            '-t', '--target',
+            '-t',
+            '--target',
             dest='target_dir',
             metavar='dir',
             default=None,
             help='Install packages into <dir>. '
-                 'By default this will not replace existing files/folders in '
-                 '<dir>. Use --upgrade to replace existing packages in <dir> '
-                 'with new versions.'
+            'By default this will not replace existing files/folders in '
+            '<dir>. Use --upgrade to replace existing packages in <dir> '
+            'with new versions.',
         )
         cmd_opts.add_option(
             '--user',
             dest='use_user_site',
             action='store_true',
             help="Install to the Python user install directory for your "
-                 "platform. Typically ~/.local/, or %APPDATA%\\Python on "
-                 "Windows. (See the Python documentation for site.USER_BASE "
-                 "for full details.)")
+            "platform. Typically ~/.local/, or %APPDATA%\\Python on "
+            "Windows. (See the Python documentation for site.USER_BASE "
+            "for full details.)",
+        )
         cmd_opts.add_option(
-            '--no-user',
-            dest='use_user_site',
-            action='store_false',
-            help=SUPPRESS_HELP)
+            '--no-user', dest='use_user_site', action='store_false', help=SUPPRESS_HELP
+        )
         cmd_opts.add_option(
             '--root',
             dest='root_path',
             metavar='dir',
             default=None,
-            help="Install everything relative to this alternate root "
-                 "directory.")
+            help="Install everything relative to this alternate root " "directory.",
```

Comment:
Another broken string concatenation. (It's not going to behave wrongly, just very bad formatting).